src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasInMemoryService.kt

```diff
@@ -76,11 +84,65 @@ class CosmasInMemoryService : CosmasGrpc.CosmasImplBase() {
         responseObserver.onCompleted()
     }
 
+    private fun verifyGetHistoryOfVersionRequest(request: CosmasProto.GetHistoryOfVersionRequest): Status {

```

Comment:
Why is it placed here, far apart from the calling method and right in-between two other methods which are not related to this one but related to each other? 

Those who will read your code will expect related things to be grouped together -- that's natural expectation of any programmer. When programmer sees such interleaving, his WTF/second rate grows.